src/tools/driver/cmd_line_utils.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2014, the Fletch project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE.md file.
+
+library driver.cmd_line_utils;
+
+int getPortNumberFromArg(String arg, String helpText) {
```

Comment:
Maybe it would be cleaner if this helper took a closure like this:

```dart
int getPortNumberFromArgument(String argument, int onError(String issue)) {
  final int portNumber = int.parse(argument, onError: (source) {
    return onError("Required argument <port number> was not a number.");
  });
  ...
}
```